### PR TITLE
Key aliases rebased

### PR DESCRIPTION
**Pull request by jpmoral**
Please see #68 for discussion.  I couldn't update that branch for some reason so I opened this one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,46 @@ end
...
+class AnAction
+  extend LightService::Action
+  promises :my_key
+
+  executed do |context|
+    context.my_key = "value"
+  end
+end
+
+class FooAction
```
Comment (adomokos) [inline README.md]: This still says `FooAction`.

**Comment by adomokos** [comment]
This looks awesome! Thank you for it! :heart:

File: README.md (hunk 1 above)
Comment (jpmoral) [inline README.md]: Lol.  I'll fix it.

**Comment by jpmoral** [comment]
Thanks!  LightService is awesome, we use it very heavily at work.

**Comment by adomokos** [comment]
That's awesome! Thank you for contributing!

I am considering giving a talk on it as it approaches 1.0. Would you be open to a brief conversation on how you guys are using it? I am thinking Skype or Hangout. I'd like to incorporate other folks' story into my talk. Cheers!

**Comment by jpmoral** [comment]
Sure.  I'm also planning to write a blog post about contributing to open source and use this PR as an example.

**Comment by adomokos** [comment]
I love that. If there is a gem or OSS I heavily rely on, I always try to pay back by contributing to it. Even with just tiny bits like fixing documentation. We just need to pay it forward! :+1:

**Pull request merged**